app/controllers/application_controller.rb

```diff
@@ -2,14 +2,19 @@ class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
 
   helper_method :cart
-  helper_method :current_user
+  helper_method :current_user, :current_admin?
```

Comment:
Added a .current_admin? helper method that goes along with the '/public/404' functionality